src/FreeAge/client/render_window.cpp

```diff
@@ -360,6 +361,7 @@ bool RenderWindow::LoadResources() {
   auto& unitTypes = ClientUnitType::GetUnitTypes();
   unitTypes.resize(static_cast<int>(UnitType::NumUnits));
   for (int unitType = 0; unitType < static_cast<int>(UnitType::NumUnits); ++ unitType) {
+    Timer timer("LoadResources() - Load unit");
```

Comment:
I committed this by accident, but I think it could left in...

Reply:
Yes, these are probably interesting statistics.

(Note that the whole effective loading time could be significantly reduced if the loading would already be done while the lobby dialog is being shown. There are of course also other possible optimizations, such as doing disk access and pre-processing in parallel as much as possible, or loading only those graphics that are actually required for the current game situation.)